core-bundle/tests/Functional/app/config/config_test.yaml

```diff
@@ -82,17 +82,31 @@ webauthn:
                 require_resident_key: true
                 user_verification: !php/const Webauthn\AuthenticatorSelectionCriteria::USER_VERIFICATION_REQUIREMENT_REQUIRED
                 resident_key: !php/const Webauthn\AuthenticatorSelectionCriteria::RESIDENT_KEY_REQUIREMENT_REQUIRED
+        contao_frontend:
+            rp:
+                name: Contao Open Source CMS
+            authenticator_selection_criteria:
+                require_resident_key: true
+                user_verification: !php/const Webauthn\AuthenticatorSelectionCriteria::USER_VERIFICATION_REQUIREMENT_REQUIRED
+                resident_key: !php/const Webauthn\AuthenticatorSelectionCriteria::RESIDENT_KEY_REQUIREMENT_REQUIRED
     request_profiles:
         contao_backend:
             user_verification: !php/const Webauthn\AuthenticatorSelectionCriteria::USER_VERIFICATION_REQUIREMENT_REQUIRED
+        contao_frontend:
+            user_verification: !php/const Webauthn\AuthenticatorSelectionCriteria::USER_VERIFICATION_REQUIREMENT_REQUIRED
     controllers:
         enabled: true
         creation:
             contao_backend_add_authenticator:
                 options_path: '%contao.backend.route_prefix%/webauthn/add/options'
                 result_path: '%contao.backend.route_prefix%/webauthn/add/result'
```

Comment:
```suggestion
                options_path: '%contao.backend.route_prefix%/webauthn/backend/add/options'
                result_path: '%contao.backend.route_prefix%/webauthn/backend/add/result'
```

Reply:
`%contao.backend.route_prefix%` already implies this is for the backend